### PR TITLE
fixing-typos-my-first-client

### LINKED ISSUE #7368: [Bug] "Unable to get currency info for Coin1 when converting to on-chain"
# 🐛 Bug

## To reproduce
1. build diemcore
2. run examples from https://developers.diem.com/docs/core/my-first-transaction
3. note that you can not proceed after the step "account mint 0 110 Coin1" since you'll get:
==> [ERROR] Error transferring coins from faucet: Unable to get currency info for Coin1 when converting to on-chain units

**Code snippet to reproduce**
```rust
# Your code goes here
# Please make sure it does not require any external dependencies
```

**Stack trace/error message**
[ERROR] Error transferring coins from faucet: Unable to get currency info for Coin1 when converting to on-chain units
// Paste the output here
```

## Expected Behavior

<!-- A clear and concise description of what you expected to happen. -->

## System information

**Please complete the following information:**
- <!-- Diem Version --> git clone 1/23/21
- <!-- Rust Version -->
- <!-- Computer OS --> macos 11


## Additional context

Add any other context about the problem here.

### PR DESCRIPTION
**Pull request by nimishamw**
This PR contains changes to the following docs:

- My First Client: Typos only
- Wallet Admin Dashboard: Updated instructions to include login information
- My First Transaction: Closes #7368

### CODE REVIEW

File: developers.diem.com/docs/wallet-app/try-wallet-admin.md [hunk 1]
```diff
@@ -35,11 +52,13 @@ The main page of the admin UI shows the total number of registered users and the
 
 Both these pages offer a list of users. Clicking “Block” button will force the user to be logged out immediately and prevent them from entering the system again.
 
-On the administrators management page “Add administrator” button allows adding a new admin user to the system.
+On the administrators management page, the “Add administrator” button allows adding a new admin user to the system.
```
Comment (JoelMarcey) [inline developers.diem.com/docs/wallet-app/try-wallet-admin.md:55]: apostrophe?

administrator's management page

File: developers.diem.com/docs/wallet-app/try-wallet-admin.md [hunk 2]
```diff
@@ -35,11 +52,13 @@ The main page of the admin UI shows the total number of registered users and the
...
 
 
 ## Liquidity management page
 
-This page simulates management of the wallet debt for its liquidity provider. The wallet continuously sells and buys funds from the liquidity provider. While the test Diem Coins are immediately transferred, the corresponding fiat amount is accumulated and settled manually. In real-world scenarios the settlement usually involves some kind of wire transfer. Here, for demonstration purposes, clicking “Settle” marks the debt as settled immediately.
+This page simulates management of the wallet debt for its liquidity provider. The wallet continuously sells and buys funds from the liquidity provider. While the test Diem Coins are immediately transferred, the corresponding fiat amount is accumulated and settled manually. 
+
+In real-world scenarios the settlement usually involves some kind of wire transfer. In the Diem Reference Wallet, clicking “Settle” marks the debt as settled immediately (for demo purposes only).
```
Comment (JoelMarcey) [inline developers.diem.com/docs/wallet-app/try-wallet-admin.md:62]: comma

In real-world scenarios,

**Review by JoelMarcey — APPROVED** [review]
A couple of nit comments

File: developers.diem.com/docs/wallet-app/try-wallet-admin.md (hunk 1 above)
Comment (nimishamw) [reply developers.diem.com/docs/wallet-app/try-wallet-admin.md:55]: This is a page for multiple admins, so no possessive apostrophe here.

File: developers.diem.com/docs/wallet-app/try-wallet-admin.md (hunk 2 above)
Comment (nimishamw) [reply developers.diem.com/docs/wallet-app/try-wallet-admin.md]: changed and pushed to PR

**Pull request closed without merging**